404.html

```diff
@@ -26,7 +26,7 @@
           btnContainer.append(backBtn);
         }
       }
-      sampleRUM('404', { source: document.referrer });
+      sampleRUM('404', { source: document.referrer, target: window.location.href });
```

Comment:
```suggestion
      sampleRUM('404', { source: document.referrer });
```

Reply:
The `target` is redundant with the URL, no need to send it twice.

Reply:
Humm.. @chicharr didn't we check we get a referer and the current page url... ?

Reply:
I just checked. `document.referrer` is there when there is a referer and yes, of course,  `target` is a duplicated of  `cdn.url` we are tracking automatically.